plugins/epos/plugin.py

```diff
@@ -458,8 +450,56 @@ def rda_a1_04d(self):  # This one needs to improve
             msg = msg2
         return (points, msg)
 
+    @ConfigTerms(term="terms_access")
+    def rda_a1_05d(self):
+        """Indicator RDA-A1-05M
+        This indicator is linked to the following principle: A1: (Meta)data are retrievable by their
+        identifier using a standardised communication protocol. More information about that
+        principle can be found here.
+        The indicator refers to automated interactions between machines to access digital objects.
+        The way machines interact and grant access to the digital object will be evaluated by the
+        indicator.
+
+        -------
+        points
+            0 -  If the URLs are broken or they are not download URLs
+
+            100 - If the URLs are download URLs
+        """
+        points = 0
+        msg = "No data access method was found in the metadata"
+        msg2 = ""
+
+        _elements = [
+            "downloadURL",
+        ]
+        data_access_elements = self.terms_access_metadata.loc[
+            self.terms_access_metadata["element"].isin(_elements)
+        ]
+        url = self.terms_access_metadata.loc[
+            self.terms_access_metadata["element"] == "downloadURL", "text_value"
+        ]
+
+        url_list = url.values
+
+        if len(url_list) > 0:
+            msg = "Data acquisition could not be guaranteed "
+            for i in url:
+                if check_link(address):
+                    headers = requests.head(i).headers
```

Comment:
headers shall be checked for the HTTP response, here you are checking headers of a string (URL) so it will fail. You need to get the header from the HTTP response you got through `check_link()` (so this method needs to return the response first)

Reply:
Removed because it did not work as intended